android/app/src/main/java/org/uwaterloo/subletr/pages/account/AccountPageView.kt

```diff
@@ -1,15 +1,296 @@
 package org.uwaterloo.subletr.pages.account
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.imePadding
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rxjava3.subscribeAsState
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.components.button.PrimaryButton
+import org.uwaterloo.subletr.components.button.SecondaryButton
+import org.uwaterloo.subletr.components.dropdown.UnderlinedDropdown
+import org.uwaterloo.subletr.components.switch.PrimarySwitch
+import org.uwaterloo.subletr.components.textfield.UnderlinedTextField
+import org.uwaterloo.subletr.theme.SubletrTheme
+import org.uwaterloo.subletr.theme.secondaryTextColor
+import org.uwaterloo.subletr.theme.textOnSubletrPink
 
 @Composable
 fun AccountPageView(
 	modifier: Modifier = Modifier,
+	viewModel: AccountPageViewModel = hiltViewModel(),
+	uiState: AccountPageUiState = viewModel.uiStateStream.subscribeAsState(
+		AccountPageUiState.Loading
+	).value,
 ) {
-	Column(modifier = modifier) {
-		Text(text = "AccountPageView")
+	if (uiState is AccountPageUiState.Loading) {
+		Column(
+			modifier = modifier
+				.fillMaxSize(1.0f),
+			verticalArrangement = Arrangement.Center,
+			horizontalAlignment = Alignment.CenterHorizontally,
+		) {
+			CircularProgressIndicator()
+		}
 	}
+	else if (uiState is AccountPageUiState.Loaded) {
+		LazyColumn(
+			modifier = modifier
+				.fillMaxSize(fraction = 1.0f)
+				.imePadding(),
+			verticalArrangement = Arrangement.Top,
+			horizontalAlignment = Alignment.CenterHorizontally,
+		) {
+			item {
+				Box(
+					modifier = Modifier
+						.padding(30.dp)
+						.clip(CircleShape),
+				) {
+					Box(
+						modifier = Modifier
+							.width(75.dp)
+							.height(75.dp)
+							.background(Color.Gray),
```

Comment:
Hmm I'm seeing a lot of 10 multiple spacing and now 75. I think we should stick to using multiples of 8 for >32

Reply:
I fixed it but I feel like you should go complain to the PM and designer instead @AhbedShawarma

Reply:
Lool I'm fine with that as long as it looks good and is consistent throughout